src/utils.js

```diff
@@ -79,6 +83,16 @@ export async function blobToDataUrl(blob) {
   });
 }
 
+export function getFileResourcePath(href, recode = false) {
+  const path = recode ? decodeURIComponent(encodeURIComponent(href)) : href;
+  return path
+    .replace(CC_FILE_PREFIX_OLD, "web_resources")
+    .replace(CC_FILE_PREFIX, "web_resources")
+    .replace(CC_FILE_OLDFIX_DECODED, "web_resources")
+    .replace(CC_FILE_PREFIX_DECODED, "web_resources")
+    .replace("web_resources/files", "web_resources"); // support canvas_cc
```

Comment:
It would be nice to move `web_resources` to constants as well along with `web_resources/files`.
I would also add the link to the canvas readme to this comment.

Reply:
DONE